src/__stories__/stories.tsx

```diff
@@ -108,4 +114,18 @@ storiesOf('formtron', module)
         <FormtronDebugger input={objectData} schema={objectSchema} selection={objectData._selection} />
       </div>
     );
+  })
+  .add('show', () => {
+    return (
+      <div className="formtron" style={{ width: '100%', height: '100%', position: 'relative' }}>
```

Comment:
`Box`? :P

Reply:
I just copied and pasted. I'll see if I can use Box....

Reply:
Screw it if it's not possible. We would have a bit cleaner code here if a styled component was used, but if it's not possible, I can leave with it.

Reply:
heh, that was all cruft! 😆